pkg/types/args_test.go

```diff
@@ -0,0 +1,113 @@
+package types_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestArgs(t *testing.T) {
```

Comment:
This can be removed from this file. ginkgo will programmatically generate this for you when you run `ginkgo bootstrap` from within the directory you are writing your tests. It makes a `PACKAGE_suite_test.go` file to hold this.